lib/client/src/client.rs

```diff
@@ -1825,6 +1826,15 @@ impl TransactionBuilder {
                 ),
             )
         }
+        let mut ix = self.instructions.clone();
```

Comment:
this completely hides the previous `ix`, dropping the compute budget instruction added before